data/saas/dataset/hubspot_dataset.yml

```diff
@@ -0,0 +1,545 @@
+#          - name: rssEmailUrl
+#            data_categories: [user.provided.nonidentifiable]
+#            fidesops_meta:
+#              data_type: string
+#          - name: scrubsSubscriptionLinks
+#            data_categories: [system.operations]
+#            fidesops_meta:
+#              data_type: boolean
+#          - name: slug
+#            data_categories: [user.provided.nonidentifiable]
+#            fidesops_meta:
+#              data_type: string
+#          - name: smartEmailFields
+#            data_categories: [system.operations]
+#          - name: state
+#            data_categories: [system.operations]
+#            fidesops_meta:
+#              data_type: string
+#          - name: styleSettings
+#            data_categories: [system.operations]
+#            fidesops_meta:
+#              data_type: string
+#          - name: subcategory
+#            data_categories: [system.operations]
+#            fidesops_meta:
+#              data_type: string
+#          - name: subject
+#            data_categories: [user.provided.nonidentifiable]
+#            fidesops_meta:
+#              data_type: string
+#          - name: subscription
+#            data_categories: [system.operations]
+#            fidesops_meta:
+#              data_type: integer
+#          - name: subscriptionName
+#            data_categories: [user.provided.nonidentifiable]
+#            fidesops_meta:
+#              data_type: string
+#          - name: teamPerms
+#            data_categories: [system.operations]
+#          - name: templatePath
+#            data_categories: [system.operations]
+#            fidesops_meta:
+#              data_type: string
+#          - name: transactional
+#            data_categories: [system.operations]
+#            fidesops_meta:
+#              data_type: boolean
+#          - name: unpublishedAt
+#            data_categories: [user.derived.nonidentifiable.sensor]
+#            fidesops_meta:
+#              data_type: integer
+#          - name: updated
+#            data_categories: [user.derived.nonidentifiable.sensor]
+#            fidesops_meta:
+#              data_type: integer
+#          - name: updatedById
+#            data_categories: [system.operations]
+#            fidesops_meta:
+#              data_type: integer
+#          - name: url
+#            data_categories: [user.provided.nonidentifiable]
+#            fidesops_meta:
+#              data_type: string
+#          - name: useRssHeadlineAsSubject
+#            data_categories: [system.operations]
+#            fidesops_meta:
+#              data_type: boolean
+#          - name: userPerms
+#            data_categories: [system.operations]
+#          - name: vidsExcluded
+#            data_categories: [system.operations]
+#          - name: vidsIncluded
+#            data_categories: [system.operations]
+#          - name: widgets
+#            data_categories: [system.operations]
+      - name: subscription_preferences
```

Comment:
Why do we not show that this data is nested under the `subscriptionStatuses` key?  We're also missing the recipient key which is the user email.

```json
{
    "recipient": "*******",
    "subscriptionStatuses": [
        {
            "id": "*******",
            "name": "*******",
            "description": "*******",
            "status": "*******",
            "sourceOfStatus": "*******",
            "brandId": "*******",
            "preferenceGroupName": "*******",
            "legalBasis": "*******",
            "legalBasisExplanation": "*******"
        }
    ]
}
```

Reply:
We target the `subscriptionStatuses` path in the `config.yml`, like this:

```
data_path: subscriptionStatuses
```

So then `recipient` as a key isn't needed in the `dataset.yml`. Is there a different way to do this that I'm missing?

Reply:
I just didn't understand why you chose to omit the `recipient`, then, as it is PII.  You are just showing subscriptionStatuses, and those fields are all marked as system-operations, so we're not returning anything there.

Reply:
Ah I see now, I'll update to include `recipient`